addons/web/static/src/js/fields/abstract_field.js

```diff
@@ -114,6 +114,7 @@ var AbstractField = Widget.extend({
         // the inner views...
         var fieldsInfo = record.fieldsInfo[this.viewType];
         this.attrs = options.attrs || (fieldsInfo && fieldsInfo[name]) || {};
+        this.attrs.options = Object.assign({}, options.nodeOptions);
```

Comment:
I would like not to do that (in some cases, you write inside fieldsInfo). FieldWidgets should stop reaching the nodeOptions from the attrs (actually, they couldn't if those weren't py-eval-ed, as suggested below).